composeApp/src/main/kotlin/io/github/couchtracker/App.kt

```diff
@@ -38,33 +39,35 @@ fun App() {
     MaterialTheme(colorScheme = darkColorScheme()) {
         CompositionLocalProvider(LocalNavController provides navController) {
             Surface(color = MaterialTheme.colorScheme.background) {
-                ProfilesContext {
-                    NavHost(
-                        navController = navController,
-                        startDestination = MainScreen,
-                        enterTransition = {
-                            val slideSpec = tween<IntOffset>(AnimationDefaults.ANIMATION_DURATION_MS, easing = LinearOutSlowInEasing)
-                            slideInVertically(slideSpec) { (it * ANIMATION_SLIDE).roundToInt() } +
-                                fadeIn(FADE_ANIMATION_SPEC)
-                        },
-                        exitTransition = {
-                            fadeOut(FADE_ANIMATION_SPEC)
-                        },
-                        popEnterTransition = {
-                            fadeIn(FADE_ANIMATION_SPEC)
-                        },
-                        popExitTransition = {
-                            val slideSpec = tween<IntOffset>(AnimationDefaults.ANIMATION_DURATION_MS, easing = FastOutLinearInEasing)
-                            slideOutVertically(slideSpec) { (it * ANIMATION_SLIDE).roundToInt() } +
+                TmdbLanguagesContext {
```

Comment:
General thought: what happens when we add another 100 app settings? We certainly can't have a local composition for each of them.

Handling the loading state for each of them is also unfeasible I think.